pkg/controller/cluster/main-controller.go

```diff
@@ -1141,6 +1141,11 @@ func (c *Controller) syncHandler(key string) error {
 		return err
 	}
 
+	if err := c.createBuckets(ctx, tenant, tenantConfiguration); err != nil {
+		klog.V(2).Infof("Unable to create MinIO buckets: %v", err)
+		return err
+	}
+
```

Comment:
if we return an error, it will simply retry the operator all over again (from the top)

Reply:
@harshavardhana I'm trying to help out here - what is the LDAP user here? are a LDAP user's credentials going to be given for creating default buckets?

Reply:
Creating the user using console user only. In case of LDAP the admin needs to specify some ldap user's login user credentials to login which are not present in the yaml files so, skipping the bucket creation in case of LDAP.